### PR TITLE
Remove noob bait

### PR DESCRIPTION
**Pull request by Simplehorror**
# Document the changes in your pull request
Warping if you haven't camoflauged your jumpsuit asks if you're sure

# Changelog

:cl: 
rscadd: Warping if you haven't camoflauged your jumpsuit asks if you're sure
/:cl:

### CODE REVIEW

**Comment by Sniblet** [comment]
camoflauging
camouflaging

**Review by ReddicusDragon — APPROVED** [review]
Low effort but valid PR, recommend changing pull request name from "Remove noob bait" as it is ambiguous.

File: code/game/machinery/computer/camera_advanced.dm [hunk 1]
```diff
@@ -360,6 +360,11 @@
 	if(!AR.clockwork_warp_allowed)
 		to_chat(user, "[span_sevtug_small("[AR.clockwork_warp_fail]")]")
 		return
+
+	if (user.w_uniform && user.w_uniform.name == "ratvarian engineer's jumpsuit")
```
Comment (Bibby0110) [inline code/game/machinery/computer/camera_advanced.dm]:
```suggestion
	if (user.w_uniform && user.w_uniform.name == initial(user.w_uniform.name))
```

**Review by Bibby0110 — CHANGES_REQUESTED** [review]

**Review by JamieD1 — APPROVED** [review]
Do bibby's changes

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**